src/dbsync/api.rs

```diff
@@ -2621,7 +2655,7 @@ mod tests {
         let _func_value = super::pool_ticker(dp.provider(), pool_hash).await.unwrap();
         let _manual_value = "EUSKL";
 
-        // assert_eq!(_func_value, _manual_value); // The value change in the future. But it passed at the time of this writing.
+        // assert_eq!(_func_value, _manual_value); // The value might change in the future. But it passed at the time of this writing.
```

Comment:
Why are these assertions commented out? If there are not being checked, we should remove them.

We can add them later when we need them.

Reply:
They are commented out because they will fail otherwise. Keeping them as comments allows us to more easily add them back by just uncommenting it, and it also serves as documentation to show that the unit test has passed when it was created

Reply:
Let fix or remove it not needed. We can add it back if needed.

Reply:
we might as well remove the entire unit tests then, but adding it back will be time-consuming and result in unnecessary cognitive load. Having it like it is now will allow us to easily switch the unit tests on/off as we need it